### PR TITLE
Fixing cli cloudsnaps

### PR DESCRIPTION
**Pull request by andreipope**
@harsh-px quite a few fixes

### CODE REVIEW

File: content/reference/CLI/cloud-snaps.md [hunk 1]
```diff
@@ -8,32 +8,33 @@ weight: 3
...
-After 6 incremental backups, every 7th backup is a full backup.
+The **first** backup uploaded to the cloud is always a **full backup**. After that, subsequent backups are incremental.
+After **6 incremental** backups, every **7th backup** is a **full** backup.
 
-### Restoring a PX Volume from cloud storage
+### Restoring a PX volume from cloud storage
 
-Any PX Volume backup can be restored to a PX Volume in the cluster. The restored volume inherits the attributes such as file system, size and block size from the backup. Replication level of the restored volume defaults to 1 irrespective of the replication of the volume that was backed up. Users can increase replication factor once the restore is complete on the restored volume.
+You can restore a backup of a _PX_ to one of your _PX_ volumes in the cluster. Once restored, the volume inherits the attributes from the backup (e.g.: file system, size and block size). The replication level of the restored volume defaults to 1, irrespective of the replication level of the volume that was backed up. Users can increase the replication factor once the restore is complete on the restored volume.
```
Comment (talakad) [inline content/reference/CLI/cloud-snaps.md]: Shouldn't this be "You can restore a backup of a _PX_ volume to" ?  Also it should be " to a _PX_ volume" as it cannot be restored to an an existing volume.

Comment (harsh-px) [reply content/reference/CLI/cloud-snaps.md]: Thanks for the review @talakad 

@andreipope can you make the above change?

Comment (andreipope) [reply content/reference/CLI/cloud-snaps.md]: @harsh-px I think the change was made on the old version. Anyway, I've "ported" it to the new version.

File: content/reference/CLI/cloud-snaps.md [hunk 2]
```diff
@@ -8,32 +8,26 @@ weight: 3
 
 ## Overview of cloud backups
 
-This document outlines how PX volumes can be backed up to different cloud provider's object storage including any S3-compatible object storage. If a user wishes to restore any of the backups, they can restore the volume from that point in the timeline. This enables administrators running persistent container workloads on-prem or in the cloud to safely backup their mission critical database volumes to cloud storage and restore them on-demand, enabling a seamless DR integration for their important business application data.
+This document outlines how to back-up _PX_ volumes to different cloud providers' object storage, including any S3-compatible object storage. If a user wishes to restore a specific backup, he can restore the volume from that point in time.
```
Comment (harsh-px) [inline content/reference/CLI/cloud-snaps.md]: Avoid using "he" as the user as it hints at gender bias.

Sentence can be renamed to

```
To restore a specific backup, the user can restore the volume from that point in time.
```

File: content/reference/CLI/cloud-snaps.md [hunk 3]
```diff
@@ -446,33 +580,47 @@ NAME					SOURCEVOLUME									STATE		NODE		BYTES-PROCESSED	TIME-ELAPSED	COMPLETE
...
+Only cloudsnaps which do not have any dependant cloudsnaps (ie incrementals) can be deleted. If there are dependent cloudsnaps then the command will throw an error and will show the list of cloudsnaps that need to be deleted first.
 
-An example to delete the backup `pqr9-cl1/538316104266867971-807625803401928868` is below:
+{{<info>}}
+For _PX_ versions above and including 2.1 and 1.7.8, delete requests are queued and processed in the background. Since querying cloud to figure out dependent backups can take a while, user requests to delete the backups are added to a queue and an immediate response is returned to the user. If a cloud backup could not be deleted because of other dependent backups, an alert is logged and this will be deleted when all other dependent backups are deleted by the user.
```
Comment (harsh-px) [inline content/reference/CLI/cloud-snaps.md]: @talakad is 1.7.8 released? I haven't seen a plan to release this version.

File: content/reference/CLI/cloud-snaps.md (hunk 2 above)
Comment (andreipope) [reply content/reference/CLI/cloud-snaps.md]: @harsh-px  My bad... I'll keep that in mind. Thanks!

**Pull request merged**